enterprise/frontend/src/metabase-enterprise/license/components/LicenseAndBillingSettings/LicenseAndBillingSettings.tsx

```diff
@@ -173,7 +173,7 @@ const LicenseAndBillingSettings = ({
 export default connect(
   (state: State): StateProps => ({
     settingValues: state.admin.settings.settings,
-    upgradeUrl: getUpgradeUrl(state, { utm_media: "license" }),
+    upgradeUrl: getUpgradeUrl(state, { utm_content: "license" }),
```

Comment:
There is one link in the static embed modal that calls `getUpgradeUrl` so I have to change it everywhere.